icd_windows_hkr.c

```diff
@@ -359,6 +341,8 @@ bool khrIcdOsVendorsEnumerateHKR(void)
                     &szGuid,
                     0);
 
+                KHR_ICD_ASSERT(devpropType == DEVPROP_TYPE_GUID);
```

Comment:
If the ret is not success the assert seems a bit weird, as devpropType might not get populated the way we think it should.

Reply:
Fixed